servicecontrol/servicecontrol-instances/configuration.md

```diff
@@ -532,39 +570,74 @@ Version: 5.4.0+
 
 The AWS Access Key ID to use to discover queue names and gather per-queue metrics.
 
-Type: string
+| Context | Name |
+| --- | --- |
+| **Environment variable** | `LICENSECOMPONENTS_AMAZONSQS_ACCESSKEY` |
+| **App config key** | `LicensingComponent/AmazonSQS/AccessKey` |
+
+| Type | Default value |
+| --- | --- |
+| string | existing transport connection string |
```

Comment:
I would change this text to be a bit more specific, perhaps:
```suggestion
| string | obtained from ServiceControl transport connection string |
```

Reply:
fixed